www/src/context/SettingsContext.jsx

```diff
@@ -23,11 +23,13 @@ const SettingsContextProvider = ({ children }) => {
     document.body.setAttribute('dir', e.target.value);
     setDirection(e.target.value);
     global.localStorage.setItem('pgn__direction', e.target.value);
+    window.analytics.track('Direction change');
```

Comment:
We should use `global` instead of `window` here (similar to `global.localStorage` above). Also, I think it would make sense to also include the direction (`e.target.value`) as an event property here?

Reply:
Right, added `e.target.value`